apps/server/src/maps.ts

```diff
@@ -1,6 +1,16 @@
 import * as config from './config';
+import { redis } from './redis';
 
 export async function geocodeAddress(address: string) {
+  const cachedLocation = await redis.get(`geocode:${address}`);
```

Comment:
If the redis.get call fail, it could affect functionality. Consider wrapping these calls in a try-catch block to handle any potential Redis connection errors gracefully.

Reply:
ok